seed/tests/test_analyses_views.py

```diff
@@ -20,7 +20,7 @@
 from seed.utils.organizations import create_organization
 
 
-class TestAnalyses(TestCase):
+class TestAnalysesView(TestCase):
```

Comment:
thought I'd move the file and rename the test b/c it's testing the views not the model